doc/develop/test/ztest.rst

```diff
@@ -378,7 +378,7 @@ Best practices for declaring the test suite
 *******************************************
 
 *twister* and other validation tools need to obtain the list of
-subcases that a Zephyr *ztest* test image will expose.
+test cases that a Zephyr *ztest* test image will expose.
```

Comment:
is it actually "test cases"? I'm not familar with ztest.

Reply:
yes, test cases.